packages/pluggableWidgets/carousel-web/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- We added new properties such as *autoplay*, *animation*, *pagination*, *navigation*, *infinite loop*, *delay*.
+
+### Changed
+- We changed the library for Carousel widget to use Swiper.
+- We changed the data source selection, now you can put your preferred widget inside carousel content.
```

Comment:
Should also add something along the lines of "We changed the widget to a pluggable widget". I think that also covers this point, so perhaps you can remove it then.

Reply:
I agree with adding  "We changed the widget to a pluggable widget" but don't see how it's connected with the last item. Because we could have changed it to pluggable but without using widget attribute.

Reply:
Part of being a "pluggable widget" is being able to nest widgets (= widget attribute). But if it's separate from that, then just leave it.